### PR TITLE
Fixes #116 implements butterknife

### LINKED ISSUE #116: Add butterknife for view injections
Add [Butterknife](https://github.com/JakeWharton/butterknife) for view injections.

### PR DESCRIPTION
**Pull request by asitava1998**
Although in some activity files, views have not been defined yet, Butterknife has been added as views would be added in future definitely. In these files there is an import statement and a statement Butterknife.bind(this) which would work when views are defined for these files/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,9 +3,11 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.jakewharton:butterknife-gradle-plugin:8.6.0'
```
Comment (CloudyPadmal) [inline build.gradle]: Is this file necessary?

Comment (asitava1998) [inline build.gradle]: In the documentation it was included. Is it running without this file? Is the Travis check passing? If yes, then not necessary.

Comment (viveksb007) [inline build.gradle]: Its not necessary, works fine with adding in app level gradle file.
